packages/aragon-cli/src/commands/dao_cmds/apps.js

```diff
@@ -51,6 +51,7 @@ exports.handler = async function({
 }) {
   knownApps = listApps(module ? [module.appName] : [])
   const web3 = await ensureWeb3(network)
+  let apps, daoAddress
```

Comment:
I change to use `ctx` to keep consistent with the rest of the codebase. We should discuss about best strategy on next sync